src/rsa/rsa.rs

```diff
@@ -76,19 +76,19 @@ macro_rules! rsa_pkcs1 {
     }
 }
 
-rsa_pkcs1!(RSA_PKCS1_2048_8192_SHA1, 2048, RSA_PKCS1_SHA1,
+rsa_impl!(RSA_PKCS1_2048_8192_SHA1, 2048, RSA_PKCS1_SHA1,
```

Comment:
I don't understand the point of this change unless it is supposed to be useful after we add PSS. If so, then let's move it to where PSS is added. Also, the indention of the text is wrong.